contents/blog/missing-recordings.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2021-06-10
+title: The mystery of missing recordings at PostHog
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/
+featuredImageType: full
+author: ["rick-marron"]
+categories: ["Engineering"]
+---
+
+[Session Recording](product/session-recording) is one of PostHog's core features. It allows customers to see their website being used, mouse clicks and all. It's really cool, but, until recently, we had one big issue with the feature. Customers would often tell us recordings were missing, and we didn't know why.
```

Comment:
Am I correct in thinking the feature was still in beta when this problem was highlighted? We should say that if it's the case.

Reply:
Good call - snuck in a line about moving it out of beta.